src/ralph/accessories/models.py

```diff
@@ -0,0 +1,142 @@
+import reversion
+from dj.choices import Choices
+from django import forms
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from ralph.accounts.models import RalphUser, Regionalizable
+from ralph.back_office.models import Warehouse
+from ralph.lib.mixins.models import AdminAbsoluteUrlMixin, TimeStampMixin
+from ralph.lib.transitions.decorators import transition_action
+from ralph.lib.transitions.fields import TransitionField
+from ralph.lib.transitions.models import TransitionWorkflowBaseWithPermissions
+
+
+class AccessoriesStatus(Choices):
+    _ = Choices.Choice
+
+    new = _('new')
+    in_progress = _('in progress')
+    lost = _('lost')
+    damaged = _('damaged')
+    used = _('in use')
+    free = _('free')
+    return_in_progress = _('return in progress')
+    liquidated = _('liquidated')
+    reserved = _("reserved")
+
+
+class Accessories(
```

Comment:
We have a `DataCenterAsset` model, not `DataCenterAssets`. I would stick to singular form here to be consistent, so this would be `class Accessory`